client/modules/User/components/DashboardTabSwitcher.jsx

```diff
@@ -57,6 +79,51 @@ const DashboardTabSwitcher = ({ currentTab, isOwner, username }) => {
           </Tab>
         )}
       </div>
+      <MediaQuery maxWidth={770}>
+        {(mobile) => (
+          <FilterOptions className={mobile ? 'visible' : 'hidden'}>
+            <div>
+              <IconButton icon={FilterIcon} />
+              <ul>
+                <li>
+                  <button
+                    onClick={() => dispatch(toggleDirectionForField('name'))}
+                  >
+                    Name
+                  </button>
+                </li>
+                <li>
+                  <button
+                    onClick={() =>
+                      dispatch(toggleDirectionForField('createdAt'))
+                    }
+                  >
+                    Created
+                  </button>
+                </li>
+                <li>
+                  <button
+                    onClick={() =>
+                      dispatch(toggleDirectionForField('updatedAt'))
+                    }
+                  >
+                    Updated
+                  </button>
+                </li>
+                <li>
+                  <button
+                    onClick={() =>
+                      dispatch(toggleDirectionForField('numItems'))
+                    }
+                  >
+                    # of sketches
```

Comment:
We need to show this one only when viewing the collections list, as it doesn't make sense or do anything on the sketches list.